.cirrus.yml

```diff
@@ -0,0 +1,63 @@
+env:
+  CIRRUS_CLONE_DEPTH: 1
+
+fedora_30_task:
+  container:
+    image: quay.io/avocado-framework/avocado-vt-ci-fedora-30
+  env:
+    PYTHON: python3
+    matrix:
+      - AVOCADO_SRC: avocado-framework<70.0
+      - AVOCADO_SRC: -e git+https://github.com/avocado-framework/avocado#egg=avocado_framework
+    matrix:
+      - SETUP: setup.py develop --user
+      - SETUP: -m pip install .
+  setup_script: &setup_scr
+    - $PYTHON --version
+    - $PYTHON -m pip install $AVOCADO_SRC
+    - $PYTHON -m pip install -r requirements.txt
```

Comment:
The real test would be not to install any dependencies in advance (with the exception of avocado==master, which needs to be installed in advance). Even the requirements.txt should be optional and the basic set of tests should work without them.

Reply:
What I meant is something like:

```
if $AVOCADO_SRC; then $PYTHON -m pip install $AVOCADO_SRC; fi
```

and using matrix:

```
- AVOCADO_SRC: ""
- AVOCADO_SRC: -e git+https://github.com/avocado-framework/avocado#egg=avocado_framework
```

Reply:
Thanks, btw that's currently the only (minor) issue (and hopefully it works well as it didn't used to work)